modules/create-resource-init-bundle.adoc

```diff
@@ -36,17 +36,17 @@ endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="create-resource-init-bundle_{context}"]
-= Creating resources by using the init bundle
+= Applying the init bundle on the secured cluster
 
 //Do not show for ACSCS
 ifndef::cloud[]
-Before you install secured clusters, you must use the init bundle to create the required resources on the cluster that will allow the services on the secured clusters to communicate with Central.
-endif::[]
+Before you install secured clusters, you must apply the init bundle by using it to create the required resources on the cluster that will allow the services on the secured clusters to communicate with Central.
```

Comment:
Given the heading **on the secured cluster** here are questions as a reader:
1. Before you install **secured clusters** (plural) means install **secured cluster services**?
2. required resources on **the** cluster: this is the to-be-secured cluster?
3. the services on the **secured clusters** (plural) would singular make it clearer it is same cluster as item 2?

Reply:
Rewriting to avoid the awkwardness, making cluster singular was a good suggestion!